README.md

```diff
@@ -101,6 +104,30 @@ class Aircraft(models.Model):
 | `QuerySet.update()`           | No
 | `QuerySet.update_or_create()` | Yes (via `QuerySet.get_or_create()` and `Model.save()`)
 
+#### Auditing Special QuerySet Writes
+
+Auditing for the four "special" QuerySet methods that perform DB writes (labeled
+**No** in the table above) _can_ be enabled. This requires three extra usage
+details:
+
+1. Enable the feature by calling the audit decorator specifying
+   `@audit_fields(..., audit_special_queryset_writes=True)`.
+2. Configure the model class so its default manager is an instance of
+   `field_audit.models.AuditingManager`.
```

Comment:
Guessing that in the code context where this framework uses this, it can't infer it from `audit_special_queryset_writes=True` above (otherwise it's be nice to only have to do one of these). Am I following right?

Reply:
The convention is to configure all auditing behavior with the decorator. It would be nice if the decorator could configure the correct manager _for you_, but managers are a very configurable feature in themselves, and having a decorator do that would be ugly and unconventional at best. If a customer desires "special queryset write auditing", and bungles the manager somehow, they will never get any errors that tell them "you're not getting the audit records you're expecting", so this decorator kwarg is a case of _"I want to use this feature. Tell me if I'm doing it wrong"_.

Reply:
I see, that helps me understand the code a little better. If you have `audit_special_queryset_writes=True` and you _don't_ configure the model class so its default manager is an instance of `AuditingManager`, then it will raise `InvalidManagerError` on startup. Might be a note worth including here, but also easy for someone to figure out by using it.

Reply:
I'll add that, thanks.